packages/bpk-tokens/src/ios/base/borders.json

```diff
@@ -4,6 +4,18 @@
     "type": "size"
   },
   "props": {
+    "BORDER_SIZE_SM": {
```

Comment:
I previously removed these as they're no longer used by native iOS, but I didn't realise that removing them would remove them from RN where we need to still use them.